content/other_applications/member_check.md

```diff
@@ -12,15 +12,15 @@ The member check can be found at [utn.se/medlemskoll](https://www.utn.se/medlems
 ## API
 
 The member check is also available as an api and can be used from a website, app etc.
-To use the api, send a **POST** request to `https://old.utn.se/member_check/check_membership.php`.
+To use the api in Moore, send a **POST** request to `/member_check_api/`.
```

Comment:
You need to specify the whole link so that users outside of moore can use the API as well
```suggestion
To use the api, send a **POST** request to `https://utn.se/member_check_api/`.
```